--- a/include.mk
+++ b/include.mk
@@ -3,6 +3,10 @@
 # build-time configuration should be done in config.mk
 # see the config-sameple*.mk files for examples
 
+prefix = usr/
+bindir = $(prefix)/bin
+sharedir = $(prefix)/share
+
 HOSTPLATFORM:=$(patsubst MINGW%,MINGW,$(shell uname -s))
 ifeq ($(HOSTPLATFORM),MINGW)
 OSTYPE=Windows
@@ -117,3 +121,7 @@
 		$(MAKE) -C $$i FOLDER="$(FOLDER)$$i/" cleand || exit 1; \
 	done
 
+install: all
+	install chdkptp $(DESTDIR)$(sharedir)/chdkptp/
+	install -Dm755 chdkptp.sh $(DESTDIR)$(bindir)/chdkptp
+	cp -r lua $(DESTDIR)$(sharedir)/chdkptp/
